ui-tests-protractor/e2e/page-objects/integrations/edit/edit.po.ts

```diff
@@ -48,7 +48,7 @@ export class FlowViewComponent implements SyndesisComponent {
   }
 
   async getStepsArray(): P<any> {
-    const stepFactory = new StepFactory();
+    const stepComponentFactory = new StepFactory();
     const steps = this.rootElement().all(by.className(FlowViewComponent.stepSelector));
```

Comment:
I would prefer separate PR to address Typescript code in the future :)

Reply:
👍